### DISCUSSION TITLE
Change the error message when trying to remove an item that has a relationship with another item

### DISCUSSION THREAD

**Discussion by LuizHAP** [directus/directus#11659] [Feature Requests]
I'm trying to change the message when deleting an item that is related to another item. Below the error message that appears and what is the relationship of the collection (Many to One)
 
```js
import { defineHook } from '@directus/extensions-sdk';

export default defineHook(({ filter }, { exceptions }) => {
  const { UnprocessableEntityException } = exceptions;

  filter('request.error', error => {
    throw new UnprocessableEntityException(
      'A data de agendamento deve ser maior que data de hoje'
    );
  });
});
```

![Captura de tela de 2022-02-15 17-50-54](https://user-images.githubusercontent.com/20971507/154147073-ea64f0d1-dc1c-49d4-9910-146c9d5e9186.png)
![Captura de tela de 2022-02-15 17-49-26](https://user-images.githubusercontent.com/20971507/154147076-f827c922-b382-42c1-b050-3301bead0392.png)

**Comment by rijkvanzanten** [comment]
I'm sorry, but is this a question or an example? Is this something that's already implemented? The category you chose confuses me a little 😅
